lxqt-config-monitor/managesavedsettings.cpp

```diff
@@ -164,6 +167,14 @@ void ManageSavedSettings::onApplyItem()
     if (ui.allConfigs->currentItem() == nullptr)
         return;
     MonitorSavedSettings settings = ui.allConfigs->currentItem()->data(Qt::UserRole).value<MonitorSavedSettings>();
+
+    if (not isHardwareCompatible(settings)) {
```

Comment:
Please change this to `if (!isHardwareCompatible(settings)) {`. In LXQt we don't use `and`, `or` or `not`.

Reply:
Sure.